hack/images/dcos/README.md

```diff
@@ -3,7 +3,6 @@
 ## build and push the docker image
 :; make push TARGET=$HOME/bin/km
 ```
-
```

Comment:
Could you add a high-level summary here about SkyDNS, kube2sky, and how it all interacts (or doesn't) with Mesos-DNS?